doc/userguide/rules/payload-keywords.rst

```diff
@@ -414,49 +414,53 @@ When ``relative`` is included, there must be a previous ``content`` or ``pcre``
 The result can be stored in a result variable and referenced by
 other rule options later in the rule.
```

Comment:
This should be moved together cf https://github.com/OISF/suricata/pull/7706#discussion_r954921598 right ?